scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the statistics (mean, std) of scalar maps, which can represent
+diffusion metrics, in a ROI.
+
+The mask can either be a binary mask, or a weighting mask. If the mask is
+a weighting mask it should either contain floats between 0 and 1 or should be
+normalized with --normalize_weights.
+
+IMPORTANT: if the mask contains weights (and not 0 and 1 exclusively), the
+standard deviation will also be weighted.
+"""
+
+import argparse
+import glob
+import json
+import logging
+import nibabel as nib
```

Comment:
So close to having it all :) Nibabel and Numpy are 3rd party, should go in second block.